.github/workflows/debian.yaml

```diff
@@ -38,15 +38,16 @@ jobs:
         run: |
           python3 -m venv .venv && . .venv/bin/activate && \
           python3 -m pip install --upgrade pip && \
-          pip3 install -r requirements.txt -r test_requirements.txt
+          python -m pip install -r requirements.txt -r test_requirements.txt
 
       - name: Setup softhsm2
         run: |
           make new_softhsm
 
       - name: Test
         run: |
-          . .venv/bin/activate && make test
```

Comment:
Maybe comment out line 49 and a comment "https://github.com/pyauth/python-pkcs11/issues/171"